pkgs/applications/virtualization/docker/default.nix

```diff
@@ -75,6 +75,10 @@ rec {
 
       extraPath = optionals (stdenv.isLinux) (makeBinPath [ iproute2 iptables e2fsprogs xz xfsprogs procps util-linux git ]);
 
+      postPatch = ''
+        patchShebangs .
```

Comment:
generally we like for patchshebangs to be more targeted, as it can be quite expensive for large directories.